File: app/src/main/java/com/karigor/friends/model/Picture.java
package com.karigor.friends.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Picture  implements Serializable {
    @SerializedName("large")
    @Expose
    private String large;
    @SerializedName("medium")
    @Expose
    private String medium;
    @SerializedName("thumbnail")
    @Expose
    private String thumbnail;

    public String getLarge() {
        return large;
    }

    public String getMedium() {
        return medium;
    }

    public String getThumbnail() {
        return thumbnail;
    }
}

File: app/src/main/java/com/karigor/friends/model/Picture.kt
package com.karigor.friends.model

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose
import java.io.Serializable

class Picture : Serializable {
    @SerializedName("large")
    @Expose
    val large: String? = null

    @SerializedName("medium")
    @Expose
    val medium: String? = null

    @SerializedName("thumbnail")
    @Expose
    val thumbnail: String? = null
}